### PR TITLE
vo part of software-3345

### PR DESCRIPTION
**Pull request by yzz971115**

### CODE REVIEW

File: src/webapp/project_reader.py [hunk 1]
```diff
@@ -23,6 +25,7 @@ def get_projects(indir="../projects", strict=False):
     projects = []
 
     mapping = load_yaml_file(os.path.join(indir, "_CAMPUS_GRIDS.yaml"))
+    VOsData = get_vos_data("../virtual-organizations", None)
```
Comment (matyasselmeci) [inline src/webapp/project_reader.py]:
```suggestion
    vos_data = get_vos_data(os.path.join(indir, "../virtual-organizations"), None)
```

File: src/webapp/project_reader.py [hunk 2]
```diff
@@ -38,6 +41,10 @@ def get_projects(indir="../projects", strict=False):
                 name = data['Sponsor']['CampusGrid']['Name']
                 ID = mapping[name]
                 data['Sponsor']['CampusGrid'] = OrderedDict([("ID", ID), ("Name", name)])
+            elif 'VirtualOrganization' in data['Sponsor']:
+                name = data['Sponsor']['VirtualOrganization']['Name']
+                ID = VOsData.vos[name]['ID']
```
Comment (matyasselmeci) [inline src/webapp/project_reader.py]:
```suggestion
                ID = vos_data.vos[name]['ID']
```

**Comment by matyasselmeci** [comment]
Also, Python convention is to only capitalize constants and class names, so local variables should be all lowercase.

**Comment by brianhlin** [comment]
@yzz971115 for further reading about Python conventions, check out PEP8:

- https://www.python.org/dev/peps/pep-0008/#constants
- https://www.python.org/dev/peps/pep-0008/#class-names
- https://www.python.org/dev/peps/pep-0008/#function-and-variable-names

**Comment by matyasselmeci** [comment]
This looks good, but I'm going to hold off on merging it until I get back from vacation on Thursday.

**Comment by brianhlin** [comment]
Labeling `do-not-merge` for the time being

**Pull request merged**